laravel/resources/views/banners.blade.php

```diff
@@ -54,7 +56,11 @@
     <div class="row">
         <div class="col-lg-12">
             <div class="alert alert-primary" role="alert">
-                There are no Banners configured! <button class="btn btn-link p-0" type="button" data-bs-toggle="modal" data-bs-target="#addBanner">Add now a new Banner</button>
+                @if($instance_list->count() != 0)
```

Comment:
Makes more sense, doesn't it? 😅 
```suggestion
                @if($instance_list->count() > 0)
```